schemas/contact.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require("joi");
+
+const contactSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().email(),
+  number: Joi.number().required(),
+});
```

Comment:
```suggestion
const contactSchema = Joi.object({
  name: Joi.string().required(),
  email: Joi.string().email(),
  number: Joi.number().required(),
});
``` Виносьте схеми валідації окремо e.g. /utils/validation/contactValidationSchemas.js